run_create_imagebuilder_gendo.py

```diff
@@ -204,7 +204,12 @@ def run_create_image_builder(options):
             "region": "ap-northeast-2",
             "amiDistributionConfiguration": {
                 "name": "Gendo_{{ imagebuilder:buildDate }}",
+                "amiTags": {"gendo_git_hash": git_hash_app.decode('utf-8').strip()},
```

Comment:
@GoldieKMG // git_hash 포맷은 통일성을 위해 기존 컨벤션을 지켜주세요. 그리고 Tag의 value 값에 hash 정보만 나오게 할수 없나요?
<img width="978" alt="Screen Shot 2022-10-13 at 12 27 33" src="https://user-images.githubusercontent.com/1650254/195492469-0815eec9-82d3-4c81-adc8-1af236164ddc.png">

```suggestion
                "amiTags": {f"git_hash_{name}": git_hash_app.decode('utf-8').strip()},
```

Reply:
@yoonjs2 // 제상님 원하시는 부분대로 나오게 했는데 hash 내용을 변경하기 이전에 작업했던걸로 올라갔습니다 요청주신 부분 그대로 반영될 예정입니다 :) 해시키 작업 전 이미지가 남게 해 혼선드려 죄송합니다.